zcash_client_memory/src/error.rs

```diff
@@ -65,10 +66,19 @@ pub enum Error {
     #[cfg(feature = "transparent-inputs")]
     #[error("Requested gap limit {1} reached for account {0:?}")]
     ReachedGapLimit(AccountId, u32),
+    #[cfg(feature = "transparent-inputs")]
+    #[error("Transparent derivation: {0}")]
+    TransparentDerivation(bip32::Error),
     #[error("Address Conversion error: {0}")]
     ConversionError(ConversionError<&'static str>),
 }
+#[cfg(feature = "transparent-inputs")]
 
+impl From<bip32::Error> for Error {
```

Comment:
Could this be reimplemented using the `#[from]` attribute macro? e.g. 
```
#[error("Transparent derivation: {0}")]
    TransparentDerivation(#[from] bip32::Error)
```
or does bip32::Error fail to implement some traits this requires

Reply:
Yeah, it doesnt have some traits.